README.md

```diff
@@ -71,6 +70,7 @@ The table below outlines what version of Nexus Repository the plugin was built a
 | Plugin Version | Nexus Repository Version |
 |----------------|--------------------------|
 | v0.0.2         | 3.8.0-02                 |
+| v1.0.0         | 3.21.0+                  |
```

Comment:
Please check how it is for r. **In product**
It was a suggestion from Joseph [link](https://github.com/sonatype-nexus-community/nexus-repository-r)

Reply:
Done 018a1dddbd5a3ffd0dc188c53c6f337b1e861677